spec/factories/assignments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :assignment do
+    user nil
```

Comment:
user = relationship
offspring should be user.offspring
shift = relationship

This creates an invalid assignment